usagov/vote-gov-drupal
# VOTE-326 - 328: Implement NVRF API & Tome Export

## Jira ticket

[VOTE-326](https://cm-jira.usa.gov/browse/VOTE-326) - Configure NVRF API Endpoints
[VOTE-327](https://cm-jira.usa.gov/browse/VOTE-327) - Configure NVRF API data structure
[VOTE-328](https://cm-jira.usa.gov/browse/VOTE-328) - Implement Tome Export of API Json Files
[VOTE-481](https://cm-jira.usa.gov/browse/VOTE-481) - Add AB Message block type to API

## Description

Implement NVRF API endpoints and update TOME to export API json files.

## Deployment and testing

### Pre-deploy

N/A

### Post-deploy

1. Locally run `lando drush retune`

### QA/Test

1. Visit the following endpoints from the CMS Site to confirm the delivery of correct data:
`/es/api/nvrf/states/data.json`
`/es/api/nvrf/cards/data.json`
`/es/api/nvrf/nvrf-fields/data.json`
`/es/api/nvrf/ab-messages/data.json`

2. Confirm that the STATIC Site has generated the api endpoints following the pattern below:
English/Default Language - /api/nvrf/.../data.json
Translated - /{langcode}/api/nvrf/.../data.json

## Checklist for the Developer




## Checklist for the Peer Reviewers
